parsec/core/gui/workspace_sharing_widget.py

```diff
@@ -270,10 +286,11 @@ def on_role_changed(self, user_info, role):
             user_fs=self.user_fs,
             workspace_fs=self.workspace_fs,
             user_info=user_info,
-            role=role,
+            role=None if role == NOT_SHARED_KEY else role,
```

Comment:
bugfix ?

Reply:
It was an attempt to fix a typing problem I had. I reverted theses changes.